app/models/manageiq/providers/ibm_cloud/power_virtual_servers/storage_manager/cloud_volume.rb

```diff
@@ -113,10 +113,11 @@ def self.validate_create_volume(ext_management_system)
 
   def self.raw_create_volume(ext_management_system, options)
     volume = nil
+    volume_params = nil
     ext_management_system.with_provider_connection(:service => 'PCloudVolumesApi') do |api|
       volume_params = IbmCloudPower::CreateDataVolume.new(
         'name'            => options['name'],
-        'size'            => options['size'] / 1.0.gigabyte,
+        'size'            => options['size'].to_i / 1.0.gigabyte,
```

Comment:
Note: The value here ends up as a float. The Power Cloud backend will accept non-integer volume size values but will floor them into an integer. In the future we'd like to enhance the UI to provide input directly as an integer GB value. See the following discussion for details: https://github.com/ManageIQ/manageiq-ui-classic/pull/7399#issuecomment-781749487